README.md

```diff
@@ -9,51 +9,45 @@ _This is the maven databricks plugin, which uses the databricks rest api._
 
 [![Javadocs](http://www.javadoc.io/badge/com.edmunds/databricks-maven-plugin.svg)](http://www.javadoc.io/doc/com.edmunds/databricks-maven-plugin)
 
+## Prerequisites
 
-### Building, Installing and Running
+For Users:
+- You have a databricks account
+- You are somewhat familiar with Maven and have maven installed
+- You have an s3 bucket (we will call databricksRepo) that you will use to store your artifacts. 
+- You have AWS keys that can write to this s3 bucket
+- Databricks has access to an IAM Role that can read from this bucket.
 
-How to build the project locally:
-```mvn clean install```
-
-- Not required! Because you can build and develop without it, but you will likely want Lombok configured with your IDEA:
-https://projectlombok.org/setup/intellij
-
-How to run the project locally (if applicable):
-
-
-## Running the tests
+For Contributors:
+- You need to be able execute an integration test that will actually do things on your databricks account.
 
-```mvn clean test```
-
-
-### End-to-End testing
+## Configuring
```

Comment:
@javamonkey79 I added more information on how to configure currently.

I think any larger refactors like the maven-properties mojo you sent my way should be out side of the scope of this bug fix.

Personally, I still am not convinced that having multiple options to set the properties is a bad thing. I could see how having a system property option for databricksRepo would be a win for consistency. But maybe a separate issue?

Reply:
Yeah, it's fine to just make another issue to address it.